test/Microsoft.DotNet.Tools.Tests.Utilities/Commands/TestCommand.cs

```diff
@@ -55,32 +55,37 @@ public virtual CommandResult Execute(string args = "")
 
         public async virtual Task<CommandResult> ExecuteAsync(string args = "")
         {
-            var resolvedCommand = _command;
-
-            ResolveCommand(ref resolvedCommand, ref args);
-
-            Console.WriteLine($"Executing - {resolvedCommand} {args} - {WorkingDirectoryInfo()}");
-            
-            return await ExecuteAsyncInternal(resolvedCommand, args);
+            return await ExecuteAsyncInternal(
+                             _command, 
+                             args, 
+                             captureOutput: false);
         }
 
         public virtual CommandResult ExecuteWithCapturedOutput(string args = "")
         {
+            return Task.Run(async () => await ExecuteAsyncInternal(
+                                                  _command, 
+                                                  args, 
+                                                  captureOutput: true)).Result;
+        }
+
+        private async Task<CommandResult> ExecuteAsyncInternal(
+            string executable, 
+            string args,
+            bool captureOutput)
+        {
             var resolvedCommand = _command;
 
             ResolveCommand(ref resolvedCommand, ref args);
-
-            var commandPath = Env.GetCommandPath(resolvedCommand, ".exe", ".cmd", "") ??
+            
+            Console.WriteLine($"[TestCommand] {WorkingDirectory}$ {resolvedCommand} {args}");
+            
+            if (captureOutput)
+            { 
+                var commandPath = Env.GetCommandPath(resolvedCommand, ".exe", ".cmd", "") ??
```

Comment:
It wasn't being used. I wondered if someone was depending on a side effect so I didn't remove the call, but I can give that a try and see if any tests fail.